packages/nextjs/app/event/[eventId]/register/page.tsx

```diff
@@ -0,0 +1,177 @@
+"use client";
+
+import React, { ChangeEvent, FormEvent, useEffect, useState } from "react";
+import { useParams } from "next/navigation";
+
+interface FormData {
+  name: string;
+  email: string;
+  organization: string;
+}
+
+interface EventData {
+  id: string;
+  eventId: string;
+  eventDate: number;
+}
+
+const RegisterToEventPage: React.FC = () => {
+  const params = useParams<{ eventId: string }>();
+  const [eventData, setEventData] = useState<EventData>();
+  const [showSuccessToast, setShowSuccessToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState();
+  const [showWarningToast, setShowWarningToast] = useState(false);
+
+  useEffect(() => {
+    async function fetchEventDataByEventID(id: string) {
+      const result = await fetch("/api/event?id=" + id);
+      const resultEventData = await result.json();
+      setEventData({
+        id: resultEventData.data._id,
+        eventId: resultEventData.data.eventId,
+        eventDate: resultEventData.data.eventDate,
+      });
+    }
+    if (params?.eventId) {
+      fetchEventDataByEventID(params.eventId as string);
+    }
+  }, [params?.eventId]);
+
+  const [formData, setFormData] = useState<FormData>({
+    name: "",
+    email: "", // Initialize with empty strings
+    organization: "",
+  });
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement>) => {
+    const { name, value } = e.target;
+
+    setFormData(prevState => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!formData.name || !formData.email || !formData.organization || !eventData?.id) {
+      alert("Some info is missing.");
+      return;
+    }
+
+    const newParticipantOnEventData = {
+      ...formData,
+      eventDBId: eventData.id,
+    };
+
+    const result = await fetch("/api/event-register", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "x-api-key": process.env.API_KEY || "",
+      },
+      body: JSON.stringify(newParticipantOnEventData),
+    });
+
+    if (result.ok) {
+      const data = await result.json();
+      setToastMessage(data.message);
+
+      if (result.status === 200) {
+        setShowSuccessToast(true);
+      }
+      if (result.status === 202) {
+        setShowWarningToast(true);
+      }
+
+      setFormData({
+        name: "",
+        email: "",
+        organization: "",
+      });
+      const timeout = setTimeout(() => {
+        setShowSuccessToast(false);
+        setShowWarningToast(false);
+      }, 4000);
+      () => clearTimeout(timeout);
+    }
+  };
+  return (
+    <div className="mx-auto">
+      <form onSubmit={handleSubmit} className="rounded-md bg-gray-300/80 p-4 w-96 grid gap-4 ">
+        <div className="text-2xl font-mus text-center">
+          Registro a {/* Event_Name */}:
```

Comment:
@Markkos89 please add the `Event Name` and the `Event Date` in the registry titles

Reply:
Fixed on #45 

cc: @HabacucMX